src/BylawsText.js

```diff
@@ -37,6 +42,22 @@ export default function BylawsText({ answers }) {
         .then(() => setIsTextCopied(true))
   }
 
+  const downloadBylaws = (text, fileType, fileName) => {
+    const blob = new Blob([text], { type: fileType })
+
+    const a = document.createElement('a')
```

Comment:
I'm not totally sure about this, but I think using `document.createElement` in React is unusal. Normally you let React create all the elements for you.

Reply:
You're right, but I coudn't do it with React. This was my source: https://gist.github.com/danallison/3ec9d5314788b337b682